spec/deface/action_view_template_spec.rb

```diff
@@ -1,104 +1,105 @@
 require 'spec_helper'
 
-module ActionView
-  describe Template do
-    include_context "mock Rails.application"
-
-    describe "with no overrides defined" do
-      before(:each) do
-        @updated_at = Time.now - 600
-
-        @template = ActionView::Template.new(
-          "<p>test</p>",
-          "/some/path/to/file.erb",
-          ActionView::Template::Handlers::ERB,
-          virtual_path: "posts/index",
-          format: :html,
-          updated_at: @updated_at
-        )
-      end
-
-      it "should initialize new template object" do
-        expect(@template.is_a?(ActionView::Template)).to eq(true)
-      end
-
-      it "should return unmodified source" do
-        expect(@template.source).to eq("<p>test</p>")
-      end
+describe ActionView::Template do
+  include_context "mock Rails.application"
+
+  let(:template) { ActionView::Template.new(
+    source,
+    path,
+    handler,
+    **options,
+    **(supports_updated_at? ? {updated_at: updated_at} : {})
+  ) }
+
+  let(:source) { "<p>test</p>" }
+  let(:path) { "/some/path/to/file.erb" }
+  let(:handler) { ActionView::Template::Handlers::ERB }
+  let(:options) {{
+    virtual_path: virtual_path,
+    format: format,
+  }}
+  let(:format) { :html }
+  let(:virtual_path) { "posts/index" }
+
+  let(:supports_updated_at?) { Deface.before_6? }
+  let(:updated_at) { Time.now - 600 }
+
+  describe "with no overrides defined" do
+    it "should initialize new template object" do
+      expect(template.is_a?(ActionView::Template)).to eq(true)
+    end
 
-      it "should not change updated_at" do
-        expect(@template.updated_at).to eq(@updated_at)
-      end
+    it "should return unmodified source" do
+      expect(template.source).to eq("<p>test</p>")
+    end
 
+    it "should not change updated_at" do
+      expect(template.updated_at).to eq(updated_at) if supports_updated_at?
```

Comment:
How about explicitly skipping the spec in case `updated_at` is not supported? That makes it easier to understand what's going on in the spec.